### ISSUE TITLE
URL Field Type - Beaver Themer Can't see?

### ISSUE THREAD

**Issue by reachadam** [pods-framework/pods-beaver-themer#20]
Using 2.7 of PODS

When trying to connect to a URL field that was created with PODS, the integration can seem to find the URL field. See 
<img width="709" alt="screenshot 2017-05-31 15 31 37" src="https://cloud.githubusercontent.com/assets/20959226/26657056/b3279a9a-4616-11e7-82af-897af78d3bfa.png">

**Comment by quasel** [comment]
Hi @reachadam - would need a bit more information, first did you check your Preview Location setting? 
Top left corner of the Themer UI - because the Plugin repopulates the Fields based on the Select CPT/Post

**Comment by reachadam** [comment]
Yes, pulls in all other fields just fine, just not the URL. I can do an animated gif to make it easy.

**Comment by reachadam** [comment]
Does this help? See how when I try to connect a URL it doesn't give me the options https://db.tt/DXaTdEBWM9

**Comment by quasel** [comment]
and you have extended the post with field of type: website?

**Comment by reachadam** [comment]
There is a relationship with a CPT named reviews. Posts have a relationship field named Product. The website URL is in the Review post that I am linking the blog post too. You see the name and all other custom fields in reviews pull in fine. 

I think it may be how PODS creates a URL custom field where you are forced to have link text.

**Comment by reachadam** [comment]
Or when there is a URL, the integration doesn't give you the option to use a custom field from the relationship. It doesn't even have the relationship as an option.

**Comment by quasel** [comment]
That's strange - traversal to related items should work the same for both cases - can you share your Configuration or have a login for me? - Will Take a closer look tomorrow!

**Comment by reachadam** [comment]
Sure I will, its on a localhost so I will send you the login via FB

**Comment by quasel** [comment]
Pods Introduce an new Field Type Link - Related to https://github.com/pods-framework/pods/issues/2655

change filter to include new field Type and take care of the output!
